server/views/pages/activities/unlock-list/planned-events.njk

```diff
@@ -193,108 +203,121 @@
 
-                        {% set headRows = (headRows.push(
-                            {
-                                html: "Event details",
-                                attributes: { "aria-sort": "none", colspan: 4 },
-                                classes: 'govuk-table__header govuk-!-display-none-print ' + ('hmpps-width-40-percent' if session.unlockListJourney.showAlerts else 'hmpps-width-50-percent')
-                            },
-                            {
-                                html: "Event details",
-                                attributes: { "aria-sort": "none" },
-                                classes: 'govuk-table__header print-only ' + ('hmpps-width-30-percent' if session.unlockListJourney.showAlerts else 'hmpps-width-40-percent')
-                            },
-                            {
-                                html: "<span><span class=\"text-underline\">R</span>efused</span>",
-                                classes: 'govuk-table__header hmpps-table__header--angled-table-header checkbox-width print-only'
-                            },
-                            {
-                                html: "<span>Rest in <span class=\"text-underline\">C</span>ell</span>",
-                                classes: 'govuk-table__header hmpps-table__header--angled-table-header checkbox-width print-only'
-                            },
-                            {
-                                html: "<span><span class=\"text-underline\">U</span>nlocked</span>",
-                                classes: 'govuk-table__header hmpps-table__header--angled-table-header checkbox-width print-only'
-                            }
-                        ), headRows) %}
-
-                        {{ govukTable({
-                            attributes: {
-                                id: 'unlock-list-table',
-                                'data-module': 'moj-sortable-table'
-                            },
-                            caption: "Prisoners",
-                            classes: "fixed-layout-table alternating-row-shading",
-                            captionClasses: "govuk-visually-hidden",
-                            head: headRows,
-                            rows: rows
-                        }) }}
+                            {{ govukTable({
+                                attributes: {
+                                    id: 'unlock-list-table',
+                                    'data-module': 'moj-sortable-table'
+                                },
+                                caption: "Prisoners",
+                                classes: "fixed-layout-table alternating-row-shading",
+                                captionClasses: "govuk-visually-hidden",
+                                head: headRows,
+                                rows: rows
+                            }) }}
+                        {% elseif session.unlockListJourney.searchTerm %}
```

Comment:
This warning message is only for empty results sets that occur when searching.

There is currently no message when an unlock list is empty because there are no prisoners on the wing or when the filter combination results in an empty set. This something to look at adding in future unlock list improvements.